xbmc/interfaces/legacy/Control.cpp

```diff
@@ -261,15 +261,17 @@ namespace XBMCAddon
 
     String ControlButton::getLabel()
     {
-      if (!pGUIControl) return NULL;
+      if (pGUIControl != nullptr)
```

Comment:
Haha sorry should have been `==`

Reply:
guess I trust you too much that I just copy-pasted :)
ofc... fixed

Reply:
I've been known to lead some astray :wink: